doc/user_guide/checkers/features.rst

```diff
@@ -65,6 +65,8 @@ Basic checker Messages
   Emitted when a name is used prior a global declaration, which results in an
   error since Python 3.6. This message can't be emitted when using Python <
   3.6.
+:name-is-parameter-and-nonlocal (E0134): *Name %r is parameter and nonlocal*
```

Comment:
You're one of the very few contributors that actually generated the doc before opening a merge request, conratulation 👍  (Actually, you're probably the first one 😄 )